stratum/hal/lib/tai/tai_manager.h

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
What is the reason we need to have another abstraction layer that converts data source calls to TAI wrapper value?
In Stratum, we have abstraction which converts from "general" to "vendor-specific".
For example, trying to get SFP information from a slot/port number is general, but how to get that information is vendor-specific.
The TAI data source is already vendor-specific, it should be able to talk to the wrapper directly.

Reply:
We plan to have other features, for example, collecting statistic info (for output/input-power (min, max, average)), or subscription mechanism. All of them are supposed to be placed in `TAIManager` class.

Reply:
Ok, should be fine if you need another component to manage the state automatically.
@pudelkoM @Andrea-Campanella any thought?